### DISCUSSION TITLE
How to send messages to multiple queues simultaneously and support sending messages separately to one of them

### DISCUSSION THREAD

**Discussion by HowardNiu** [rabbitmq/rabbitmq-server#11394] [Questions]
### Is your feature request related to a problem? Please describe.

I have many users, and each user has a queue for transmitting messages. How should I set up a Exchange and Queue to meet the following requirements?
1. Support sending messages to a queue of all users simultaneously
2. Support sending messages to a single user

User queue naming example:
user_queue.user1
user_queue.user2
user_queue.user3

### Describe the solution you'd like

I'm not sure if RabbitMQ already supports this requirement. If so, please provide a solution，appreciate！

### Describe alternatives you've considered

I use TopicExchange, where the routingKey of each queue is bind to user_queue. * during bind. This way, messages can only be sent to all users and cannot be sent to a specific user

### Additional context

appreciate！

**Comment by HowardNiu** [comment]
I understand now, when I want to send one, use 'direct';
When I want to send multiple queues, use the '#' in the TopicExchange
When I want to send all the queues, use the '. *' in TopicExchange

**Comment by ankitgr8** [comment]
When we say use # in the topic exchange for multiple queue or * for all queues  .. what do we mean by that..

We use # or * in the topic binding.. when a topic is bind to a queue in the topic pattern what we define there in that we use * or # to denote single match or multiple match..
Is that what you are referring to.. or there is some thing else at exchange level..

Just wanted to understand.. since I have never used that at Exchange level
